photutils/background/background_2d.py

```diff
@@ -745,8 +745,8 @@ def _calc_coordinates(self):
         self.yx = np.column_stack([self.y, self.x])
 
         # the position coordinates used when calling an interpolator
-        ny, nx = self.data.shape
-        self.data_coords = np.array(list(product(range(ny), range(nx))))
+        nx, ny = self.data.shape
```

Comment:
This was fixed in #1104.  You'll need to rebase to pick up those changes.  The correct order is ``ny, nx``.